lib/ansible/utils/module_docs_fragments/scaleway.py

```diff
@@ -22,6 +22,11 @@ class ModuleDocFragment(object):
       - HTTP timeout to Scaleway API in seconds.
     default: 30
     aliases: ['timeout']
+  query_parameters:
+    description:
+    - List of parameters passed to the query string
```

Comment:
should be a dict of parameters and `type: dict`

Reply:
fixed